forge/forge/op/eval/forge/convolution.py

```diff
@@ -226,57 +226,47 @@ def eval(self, tensors):
         assert len(tensors) <= 3, "ConvTranspose ops should have up to three inputs (input, weight, bias)"
         assert len(tensors) >= 2, "ConvTranspose ops should have at least two inputs (input, weight)"
         t_ops = to_torch_operands(*tensors)
-
         activations = t_ops[0]
         weights = t_ops[1]
         bias = t_ops[2] if len(t_ops) == 3 else None
 
         stride = [self.stride_height, self.stride_width]
         dilation = [self.dilation_height, self.dilation_width]
         groups = self.groups
-        padding = [
-            self.padding_left,
-            self.padding_right,
-            self.padding_top,
-            self.padding_bottom,
-        ]
-
-        channel_last = self.channel_last
+        # TODO: Add support for asymmetric padding cases in convtranspose2d
+        # https://github.com/tenstorrent/tt-forge-fe/issues/665
+        if (self.padding_left == self.padding_right and self.padding_top == self.padding_bottom):
+            padding = (self.padding_top, self.padding_left)
```

Comment:
@nvukobratTT , Support for different padding for each side for convtranspose2d is yet to be given , assertion has been removed and created issue has been added as a to-do.